Microsoft.Toolkit.Sample.Forms.App/Program.cs

```diff
@@ -16,15 +16,10 @@ static class Program
         [STAThread]
         static void Main()
         {
-            using (new UI.XamlHost.XamlApplication()
-            {
-                RequestedTheme = WUX.ApplicationTheme.Light,
-            })
-            {
-                Application.EnableVisualStyles();
-                Application.SetCompatibleTextRenderingDefault(true);
-                Application.Run(new Form1());
-            }
+            System.Runtime.CompilerServices.RuntimeHelpers.RunClassConstructor(typeof(Microsoft.Toolkit.Forms.UI.XamlHost.WindowsXamlHostBase).TypeHandle);
```

Comment:
Did you ever figure out why this workaround is needed, and who is fixing it in .NET Core?

Reply:
I am following up on a separate email thread.